lib/index.js

```diff
@@ -97,6 +97,31 @@ Parsely.prototype.page = function(page) {
  */
 
 Parsely.prototype.track = function(track) {
+  var event = track.event();
+  var videoEvents = {
+    'Video Content Started'     : 'trackPlay',
+    'Video Playback Paused'     : 'trackPause',
+    'Video Playback Interrupted': 'reset'
+  };
+
+  if (videoEvents.hasOwnProperty(event) && window.PARSELY.video) {
+    var vidId = track.proxy('properties.assetId');
+    var urlOverride = track.proxy('context.page.url');
+
+    // https://www.parse.ly/help/integration/video/#video-metadata
+    // https://paper.dropbox.com/doc/Segment-Video-Spec-jdrVhQdGo9aUTQ2kMsbnx
+    var metadata = reject({
+      title: track.proxy('properties.title'),
+      pub_date_tmsp: + new Date(track.proxy('properties.airdate')),
+      image_url: track.proxy('properties.imageUrl'),
+      section: track.proxy('properties.genre'),
+      authors: [track.proxy('properties.publisher')],
+      tags: track.proxy('properties.tags')
```

Comment:
properties.tags is not spec'd. the video content events has spec'd `properties.keywords` which is an array of tags so I think we should use that.